### PR TITLE
[MIG]11.0 mig l10n nl partner salutation

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

File: l10n_nl_partner_salutation/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (NL66278) [inline l10n_nl_partner_salutation/__init__.py:2]: License line

File: l10n_nl_partner_salutation/data/res_partner_title.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
```
Comment (NL66278) [inline l10n_nl_partner_salutation/data/res_partner_title.xml]: No more need for data. noupdate attribute can be transferred to odoo element.

File: l10n_nl_partner_salutation/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+from . import res_partner_title
```
Comment (NL66278) [inline l10n_nl_partner_salutation/models/__init__.py:2]: license line

File: l10n_nl_partner_salutation/tests/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+from . import test_partner_salutation
```
Comment (NL66278) [inline l10n_nl_partner_salutation/tests/__init__.py:2]: license line

**Review by NL66278 — APPROVED** [review]
:+1:  Some comments. Not blocking. LGTM.

**Comment by NL66278** [comment]
Failures are due to https://github.com/OCA/partner-contact/pull/660#pullrequestreview-194506834 not yet merged.

**Comment by daramousk** [comment]
@lfreeke And here

**Comment by hbrunn** [comment]
@daramousk please rebase on current 11.0

**Comment by daramousk** [comment]
Hello @sbidoul ,
Perhaps this is ready to be merged?

**Comment by NL66278** [comment]
Closing and reopening to force runbot

**Comment by andreschenkels** [comment]
He Guys,
Can I help with this migration? I would love to have this module in 12.0 and 13.0.

**Comment by NL66278** [comment]
Close and open again. To rerun runbot.

**Review by hbrunn — APPROVED** [review]

**Pull request merged**